branchtesting/src/main/java/branchexample/App.java

```diff
@@ -4,12 +4,12 @@
 {
     public void aMethod( String[] args )
     {
-    	// TODO fix this
     	int i = 0;
-    	if (i == 0) {
+    	if (i != 0) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false" ([squid:S2583](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS2583))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=22&issues=AWaH3AUulB6-zLfzr5-C&open=AWaH3AUulB6-zLfzr5-C)

[ISSUE_KEY]: AWaH3AUulB6-zLfzr5-C